zest/releaser/tests/functional-with-hooks.txt

```diff
@@ -51,7 +51,12 @@ Run the prerelease script.  Note that pyroma and check-manifest have
 hooks here so they are run too, but the order may differ.  With the
 bin/test script first pyroma is run, then check-manifest.  With tox it
 is the other way around.  So we use ellipsis for that part.
+sys.dont_write_bytecode is set to True to avoid writing .pyc files so
+that check-manifest doesn't complain about differences between the sdist
+directory and the vcs.
```

Comment:
Note to @mauritsvanrees : theoretically, the .gitignore and Manifest.in combination should be fine. But we kept on running into problems with the manifest checker complaining about the exclusion of "*.pyc" files that weren't present or git complaining about uncommitted files (pyc...). Anyway, no theoretically correct combination worked, hence the "don't write bytecode" hack.